pkg/validation/internal/good_practices.go

```diff
@@ -54,6 +54,9 @@ func validateGoodPracticesFrom(bundle *manifests.Bundle) errors.ManifestResult {
 	for _, warn := range warns {
 		result.Add(errors.WarnFailedValidation(warn.Error(), bundle.CSV.GetName()))
 	}
+	for _, err := range validateCrdDescriptions(bundle.CSV.Spec.CustomResourceDefinitions) {
+		result.Add(errors.ErrFailedValidation(err.Error(), bundle.CSV.GetName()))
```

Comment:
We need return a Warning in this scenario.
```suggestion
		result.Add(errors.WarnFailedValidation(err.Error(), bundle.CSV.GetName()))
```
We cannot fail with an error because that means blocking new distributions when we should not.

Reply:
feel free to close sorted out. :-)